onehungrymind/sermon-beacon
# SB-7: Hasura Migrations

## Feature/Problem
Hasura migrations for the data model


## Solution
Create tables for 


## Areas of Impact
backend


## UI Images
N/A


## Manual Testing
- Pull down the branch, run the server, and verify that the tables for sermons, speakers, tags, media, and media_types were created.
